frontend/src/components/product/ShoppingCart.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <!-- Shopping Cart Overlay -->
-  <div class="fixed inset-0 flex justify-end bg-black bg-opacity-50 z-50"
-    v-show="shoppingCartToggle">
-    <div class="bg-white h-full w-full max-w-md shadow-lg flex flex-col">
+  <div 
+  class="w-full h-full inset-0 flex justify-end bg-gray-500 bg-opacity-40 backdrop-blur-md"
+  v-if="shoppingCartToggle">
```

Comment:
Indent this please, one tab here.

It's important to kwno the different between `v-show` and` v-if` here.
`v-show` creates the component once and shows it when the condition is true
`v-if` creates the component every time the condition is true

Make sure you are not creating the component over and over again unnecessarily.

Reply:
In this case v-if allows don't have charged all images for all navigation in the all app, instead v-show is carry that DOM or Template for the app but with v-if the component Script it's loaded you don't have to worry about the recharged the component, that's why the component loaded only one time.